_layouts/help.html

```diff
@@ -11,6 +12,9 @@
     <div class="page-content__prose grid-col-12 desktop:grid-col-8 help-article">
       <h1>{{page.title}}</h1>
       {{ page.content | replace: 'site.baseurl', site.baseurl | markdownify }}
+      {% if page.category != "verify-your-identity" %}
+        <div id="touchpoints_feedback"></div>
```

Comment:
Not blocking, but if we wanted to avoid splitting translations between `_data/[lang]/settings.yml` and `touchpoints_translations.js`, we could consider adding the localized text as a data attribute of this element:

```suggestion
        <div
          id="touchpoints_feedback"
          data-yes="{{ site.data[page.lang].feedback_form.yes }}"
          data-no="{{ site.data[page.lang].feedback_form.no }}"
        ></div>
```

Then in `touchpoints_translations.js`:

```js
const yesButton = document.querySelector('input[type=submit][value=yes]');
const noButton = document.querySelector('input[type=submit][value=no]');
const touchpointsRoot = document.getElementById('touchpoints_feedback');
if (touchpointsRoot) {
  yesButton.value = touchpointsRoot.dataset.yes;
  noButton.value = touchpointsRoot.dataset.no;
}
```

Reply:
@aduth I think this approach is really clean, but I have a question because I think I may be missing something.

When switching over to this approach, am I supposed to add yes/no values to the `_data/[lang]/settings.yml` file? As it stands right now, both Yes and No buttons do not have a value and appear as so
<img width="620" alt="Screen Shot 2021-05-24 at 5 21 19 PM" src="https://user-images.githubusercontent.com/17969963/119409869-f4f91880-bcb5-11eb-9e54-de84b05a3f14.png">

Reply:
Yeah we'd have to add those keys to `settings.yml` manually, the goal of the approach @aduth suggested was to have translations in standard places we know to look, instead of us having to look inside the JS if we needed to update the translations